packages/python-packages/apiview-copilot/src/_apiview_reviewer.py

```diff
@@ -108,15 +108,20 @@ def get_general_review_response(self, apiview: str) -> GeneralReviewResult:
             numbered_lines.append(f"{start_line_no + i + 1:4d}: {line}")
         apiview = "\n".join(numbered_lines)
 
-        prompt_path = pathlib.Path(_PROMPTS_FOLDER) / f"review_apiview_{self.language}.prompty"
-        judge_path = pathlib.Path(_PROMPTS_FOLDER) / f"review_apiview_{self.language}_judge.prompty"
+        prompt_path = (
+            pathlib.Path(_PROMPTS_FOLDER) / f"review_apiview_{self.language}.prompty"
+        )
+        judge_path = (
```

Comment:
nit: I'm assuming this is from black? In sdk for python, we use --line-length 120 which feels a bit more reasonable with monitor sizes today

Reply:
Yeah, from black.